http/handler.go

```diff
@@ -133,7 +136,19 @@ func (h *handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 	defer cancel()
 
-	req.Context = logging.ContextWithLoggable(req.Context, loggables.Uuid("requestId"))
+	reqId := loggables.Uuid("requestId")
+	req.Context = logging.ContextWithLoggable(req.Context, reqId)
```

Comment:
Opening two logging events feels a bit redundant. Does opentrace not replace logging?

Reply:
This doesn't open logging events, it is creating a loggable requestId and associating it with the context so that events logged with the context will contain the requestId loggable. We then use the requestId below as a tag on the `rootSpan`. This is a bit clunky though..

Reply:
Events may not be the right word but we're still maintaining two independent logging "contexts". This is not a viable long-term solution and feels like a hack.

Is there any reason we're not using `logging.EventBeginInContext`? Are we not exposing something we need from that? IIRC, you changed that to open opentracing spans.